storybook/stubs/shared/stores/send/TransactionStore.qml

```diff
@@ -1,3 +1,270 @@
+    }
+
+    function getTransactions() {
+        return transactions
+    }
+
+    readonly property var transactions_: ListModel {
+        id: transactions
+
+        Component.onCompleted: {
+            for (let i = 0; i < 10; i++)
+                append({
+                           to: "to",
+                           loadingTransaction: false,
+                           value: {
+                               displayDecimals: true,
+                               stripTrailingZeroes: true,
+                               amount: 3.234
+                           },
+                           timestamp: new Date()
+                       })
+        }
+    }
+
+    function findTokenSymbolByAddress() {
+        return "ETH"
+    }
+
+    function switchSenderAccount(index) {
+        selectedSenderAccount = senderAccounts.get(index)
+    }
+
+    function getNetworkShortNames(chainIds) {
+        return ""
+    }
+
+    function getShortChainIds(chainIds) {
+        let listOfChains = chainIds.split(":")
+        let listOfChainIds = []
+        for (let k =0;k<listOfChains.length;k++) {
+            for(let i=0;i<NetworksModel.allNetworks.count;i++) {
+                if(NetworksModel.allNetworks.get(i).shortName === listOfChains[k]) {
+                    listOfChainIds.push(NetworksModel.allNetworks.get(i).chainId)
+                }
+            }
+        }
+        return listOfChainIds
+    }
+
+    function setSendType(sendType) {
+        root.sendType = sendType
+    }
+
+    function setSelectedRecipient(recipientAddress) {
+        root.selectedRecipient = recipientAddress
+    }
+
+    function setSelectedAssetSymbol(symbol) {
+       root.selectedAssetSymbol = symbol
+    }
+
+    function getWei2Eth(wei, decimals) {
+        return wei/(10**decimals)
+    }
+
+    function updateRoutePreferredChains(chainIds) {
+        root.toNetworksModel.updateRoutePreferredChains(chainIds)
+    }
+
+    function toggleShowUnPreferredChains() {
+        root.showUnPreferredChains = !root.showUnPreferredChains
+    }
+
+    property string amountToSend
+    property bool suggestedRoutesCalled: false
+    function suggestedRoutes(amount) {
+        root.amountToSend = amount
+        root.suggestedRoutesCalled = true
+    }
+
+    enum EstimatedTime {
+        Unknown = 0,
+        LessThanOneMin,
+        LessThanThreeMins,
+        LessThanFiveMins,
+        MoreThanFiveMins
+    }
+
+    function getLabelForEstimatedTxTime(estimatedFlag) {
+        switch(estimatedFlag) {
+        case TransactionStore.EstimatedTime.Unknown:
+            return qsTr("~ Unknown")
+        case TransactionStore.EstimatedTime.LessThanOneMin :
+            return qsTr("< 1 minute")
+        case TransactionStore.EstimatedTime.LessThanThreeMins :
+            return qsTr("< 3 minutes")
+        case TransactionStore.EstimatedTime.LessThanFiveMins:
+            return qsTr("< 5 minutes")
+        default:
+            return qsTr("> 5 minutes")
```

Comment:
No need to mark those for translation, this is a stub

Reply:
I favour using `qsTr` to mark user-visible strings for documenting purposes, even if they won't be translated.

Reply:
That usually screws the translations teams statistics :) So they never know whether a language is 100% done (sure, they may opt to just copy&paste the English original)

Reply:
> That usually screws the translations teams statistics :) So they never know whether a language is 100% done (sure, they may opt to just copy&paste the English original)

I don't remember cases where a user-facing string didn't have to be translated. Still, I hope nobody tries to translate the storybook :)

Reply:
Right yeah, it's a different story(book) ;)